server/controller/Songs/attribute.js

```diff
@@ -0,0 +1,7 @@
+export const SONG_ATTRIBUTE = {
+    songID: `songID`,
+    songName: `songname`,
```

Comment:
I have already adjust naming convention in MySQL. Please check it and test again! @Dev01-vinamr-Ken